src/Markdig.Wpf/Renderers/Wpf/CodeBlockRenderer.cs

```diff
@@ -3,15 +3,22 @@
 // See the LICENSE.md file in the project root for more information.
 
 using System;
+using System.Windows.Documents;
 using Markdig.Syntax;
+using Markdig.Wpf;
 
 namespace Markdig.Renderers.Wpf
 {
     public class CodeBlockRenderer : WpfObjectRenderer<CodeBlock>
     {
         protected override void Write(WpfRenderer renderer, CodeBlock obj)
```

Comment:
I don't know if you use ReSharper. I try to enforce some _nullability checks_ to make the code clearer about its intent. So here for example both parameters should be decorated with `[NotNull]`.
Not critical for the PR so can be done after merging.